test/utils_shared/mock_utils.py

```diff
@@ -0,0 +1,92 @@
+from unittest.mock import create_autospec
+
+from execution_engine2.db.MongoUtil import MongoUtil
+from execution_engine2.utils.CatalogUtils import CatalogUtils
+from execution_engine2.utils.KafkaUtils import KafkaClient
+from execution_engine2.utils.SlackUtils import SlackClient
+
+from installed_clients.authclient import KBaseAuth
+
+from execution_engine2.authorization.roles import AdminAuthUtil
+from execution_engine2.utils.Condor import Condor
+from execution_engine2.sdk.EE2Constants import ADMIN_READ_ROLE, ADMIN_WRITE_ROLE
+from execution_engine2.utils.clients import ClientSet
+
+ALL_CLIENTS = [
+    KBaseAuth,
+    AdminAuthUtil,
+    Condor,
+    CatalogUtils,
+    KafkaClient,
+    MongoUtil,
+    SlackClient,
+]
+
+
+def get_client_mocks(config, config_path, *to_be_mocked):
+    """
+    Create a client set containing a mix of mocks and real implementations as needed for
+    a test.
+
+    config is the config dict from the ee2 section of the deploy.cfg.
+    config_path is the path to the configfile.
+    to_be_mocked is the classes in the client set that should be mocked, e.g. KBaseAuth, etc.
+
+    Returns a dict of the class to the the class's mock or implementation as specified in
```

Comment:
an extra `the`

Reply:
Fixed locally, will show up in a future PR.